boa_engine/src/lib.rs

```diff
@@ -1,14 +1,34 @@
-//! This is an experimental Javascript lexer, parser and compiler written in Rust. Currently, it
-//! has support for some of the language.
+//! Boa's **boa_engine** crate implements ECMAScript's standard library of builtin objects
+//! and an ECMAScript context, bytecompiler, and virtual machine for code execution.
 //!
 //! # Crate Features
 //!  - **serde** - Enables serialization and deserialization of the AST (Abstract Syntax Tree).
 //!  - **console** - Enables `boa`'s [WHATWG `console`][whatwg] object implementation.
 //!  - **profiler** - Enables profiling with measureme (this is mostly internal).
 //!  - **intl** - Enables `boa`'s [ECMA-402 Internationalization API][ecma-402] (`Intl` object)
 //!
+//! # About Boa
+//! Boa is an open-source, experimental ECMAScript Engine written in Rust for lexing, parsing and executing ECMAScript/JavaScript. Currently, Boa
+//! supports some of the [language][boa-conformance]. More information can be viewed at [Boa's website][boa-web].
+//!
+//! Try out the most recent release with Boa's live demo [playground][boa-playground].  
+//!
+//! # Boa Crates
+//!  - **boa_ast** - Boa's ECMAScript Abstract Syntax Tree.
+//!  - **boa_engine** - Boa's implementation of core langugage functionality and execution.
+//!  - **boa_gc** - Boa's Garbage Collector
+//!  - **boa_interner** - Boa's string interner
+//!  - **boa_macros** - Boa's derive macros
+//!  - **boa_profiler** - Boa's code profiler
+//!  - **boa_tester** - Boa's Test262 runner
+//!  - **boa_unicode** - Boa's unicode identifier
+//!  - **boa_wasm** - Boa's WASM binding
+//!
```

Comment:
```suggestion
//!  - **boa_profiler** - Boa's code profiler
//!  - **boa_unicode** - Boa's unicode identifier
//!
```

I think we should only mention crates that we publish on crates.io here.

Reply:
Sounds good to me. Does the rest of the about boa header in the crate documentation make sense?

Reply:
I like it.